test/parallel/test-async-wrap-constructor.js

```diff
@@ -6,14 +6,14 @@ require('../common');
 const assert = require('assert');
 const async_hooks = require('async_hooks');
 
-for (const badArg of [0, 1, false, true, null, 'hello']) {
+[0, 1, false, true, null, 'hello'].forEach((badArg) => {
   const hookNames = ['init', 'before', 'after', 'destroy', 'promiseResolve'];
-  for (const field of hookNames) {
+  hookNames.forEach((field) => {
     assert.throws(() => {
       async_hooks.createHook({ [field]: badArg });
     }, {
       code: 'ERR_ASYNC_CALLBACK',
       name: 'TypeError',
     });
```

Comment:
we should add a message to throws so we could clearly know if some badArgs not throw error.

Reply:
it would be very useful!

Reply:
@Himself65  Do you think that the change proposed for @juanarbol resolve your recommendation? 
or were you talking about a new throw?

Reply:
sure